src/autoLayer.js

```diff
@@ -0,0 +1,78 @@
+gisportal.autoLayer = {};
+
+// Gets the value of a url parameter
+gisportal.autoLayer.url_get = function(param){
```

Comment:
This is a very generic function that could well be useful in other parts of the application so placing it in the gisportal.autolayer namespace seems odd. There's a function in gisportal.utils.getURLParameter that may already do what you need - consider using this instead.